device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,82 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
```

Comment:
what will happen when we have ipv6?

Reply:
The plan is similar to Bgp we will have rsvp_ipv6_interfaces and rsvp_ipv6_lsp_endpoints to handle RSVP TE Ipv6 specification. Note we believe no such requirements is present in mid to long term(based on existing enhancement received for rsvp te).

Reply:
If we plan to be like BGP, the way that BGP named its interfaces is very clear in including ipv4_ and ipv6_ as prefix to interface.  There is an implicit assumption in this case here that rsvpInterface is meant to be ipv4.  We should be consistent with being "explicit" about it.

Reply:
We have added ipv4 suffix/prefix/modifier  to interface , lsp_interface , p2p_egress_lsps and p2p_ingress_lsps so that future parallel ipv6 objects can be "explicit"ly different as you indicate.